src/main/java/frc/robot/commands/dropping/DropSequence.java

```diff
@@ -30,28 +31,19 @@ public class DropSequence extends SequentialCommandGroup {
      */
     public DropSequence(
         BaseSwerveSubsystem swerveSubsystem, RollerSubsystem rollerSubsystem, TiltedElevatorSubsystem tiltedElevatorSubsystem,
-        ElevatorState dropHeight, double waitTime1, double outtakePower, double outtakeDuration,
-        double waitTime2, double reverseDist, double waitTime3
+        ElevatorState dropHeight, double waitTime1, double outtakePower, double outtakeDuration, double waitTime2, double backTime
     ) {
         addRequirements(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem);
 
-        Pose2d currentPos = swerveSubsystem.getRobotPosition();
-        Pose2d targetPos = currentPos.plus(new Transform2d(
-            new Translation2d(
-                -reverseDist * currentPos.getRotation().getCos(),
-                -reverseDist * currentPos.getRotation().getSin()
-            ),
-            new Rotation2d()
-        ));
-
         addCommands(
             new TiltedElevatorCommand(tiltedElevatorSubsystem, dropHeight),
             new WaitCommand(waitTime1),
             new InstantCommand(rollerSubsystem::openMotor, rollerSubsystem),
             rollerSubsystem.getOuttakeCommand(-outtakePower, outtakeDuration),
             new WaitCommand(waitTime2),
-            // FollowPathCommand.from(swerveSubsystem, currentPos, List.of(), targetPos),
-            new WaitCommand(waitTime3),
+            new InstantCommand(() -> swerveSubsystem.setDrivePowers(-.2, 0, 0, true)),
+            new WaitCommand(backTime),
+            new InstantCommand(() -> swerveSubsystem.setDrivePowers(0, 0, 0, true)),
             new TiltedElevatorCommand(tiltedElevatorSubsystem, ElevatorState.GROUND)
```

Comment:
i think this command can finish before the elevator is completely down, so the drivers can start driving again

Reply:
Changed to an InstantCommand